File: discover_ruby/github_most_popular.rb
require 'httpclient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 6e7f3146a1a9d7a12e4aceac3f3918c2fa57be6a'
}

clnt = HTTPClient.new
puts clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')

File: discover_more_python/github_most_popular.py
from urllib2 import Request, urlopen, URLError
request_headers = {
    'User-Agent': 'Holberton_School',
    'Authorization':'token 6e7f3146a1a9d7a12e4aceac3f3918c2fa57be6a'
    }

url = Request("https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc",None, request_headers)
r = urlopen(url)
response = r.read()

print response

